### DISCUSSION TITLE
warning C4530: 使用了 C++ 异常处理程序，但未启用展开语义。请指定 /EHsc

### DISCUSSION THREAD

**Discussion by Tech-Liao** [xmake-io/xmake#4948] [Q&A]
**问题**:C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.39.33519\include\ostream(779): warning C4530: 使用了 C++ 异常处理程序，但未启用展开语义。请指定 /EHsc
![Snipaste_2024-04-08_21-42-46](https://github.com/xmake-io/xmake/assets/50518556/ce8d0e69-1271-49cc-b352-836d09ecf219)
**过程**:
创建xmake项目后,使用xmake命令出现的问题
平台是Win10,采用scoop安装方式

**Comment by waruqi** [comment]
用 dev 版本试试，我记得 dev 版本应该没这个问题。https://github.com/xmake-io/xmake/releases/download/v2.8.9/xmake-dev.win64.exe

**Comment by githubnotebooks** [comment]
```lua
if is_host("windows") then
	add_cxxflags("/EHsc")
end
```

可以暂时让它闭嘴。

**Comment by star-hengxing** [comment]
https://github.com/xmake-io/xmake/discussions/4838
